tests/drm_plugin_test.sh

```diff
@@ -136,31 +126,31 @@ function gui_control_Test
   compare_command_sequence expected_cmd_seq[@] "$output" "$ID"
 }
 
-function get_available_connectors_Test
-{
-  local ID
-  export SYSFS_CLASS_DRM="$FAKE_DRM_SYSFS"
-
-  declare -a expected_output=(
-    "[local] Card1 supports:"
-    "DP"
-    "DP"
-    "HDMI"
-    "DP"
-    "[local] Card0 supports:"
-    "DP"
-    "DP"
-    "DP"
-    "HDMI"
-    "DVI"
-  )
-
-  ID=1
-  output=$(get_available_connectors 2)
-  compare_command_sequence expected_output[@] "$output" "$ID"
-}
-
-function get_supported_mode_per_connector_Test
+# function test_get_available_connectors()
+# {
+#   local ID
+#   export SYSFS_CLASS_DRM="$FAKE_DRM_SYSFS"
+
+#   declare -a expected_output=(
+#     "[local] Card1 supports:"
+#     "DP"
+#     "DP"
+#     "HDMI"
+#     "DP"
+#     "[local] Card0 supports:"
+#     "DP"
+#     "DP"
+#     "DP"
+#     "HDMI"
+#     "DVI"
+#   )
+
+#   ID=1
+#   output=$(get_available_connectors 2)
+#   compare_command_sequence expected_output[@] "$output" "$ID"
+# }
```

Comment:
Why did you comment on all these lines? Did I miss something?

Reply:
See [line 12](https://github.com/kworkflow/kworkflow/pull/312/files#diff-1fc9441d195d422eea2139627823d58b8d18a5d4c899a736cfdf2d7c300fb055L12) of this same file: this test was commented out in the suite function.

Reply:
This test seems to always fail when run on unstable. I do not know enough about it to offer a solution, but there seems to be a mismatch between the expected result and the fake connectors added for the test.

Reply:
Now I remember this test... I added it, and I noticed some instabilities. At that time, I was in a hurry, and I decided to just comment on this test and go ahead with the feature. Btw, could you remove it from the test and create an issue for that? Then, paste that test version into the issue. Same for any other test.